ubuntudesign/documentation_builder/operations.py

```diff
@@ -171,25 +171,24 @@ def parse_markdown(parser, template, filepath, metadata):
 
         metadata.update(markdown_meta)
 
-    if metadata.get('table_of_contents'):
-        toc_soup = BeautifulSoup(parser.toc, 'html.parser')
+    toc_soup = BeautifulSoup(parser.toc, 'html.parser')
 
-        # Get title list item (<h1>)
-        nav_item_strings = []
+    # Get title list item (<h1>)
```

Comment:
no it doesn't ...
Must be a old comment, can you remove or update it?

Reply:
Removed